docs/calls.md

```diff
@@ -120,20 +124,31 @@ OPTIONS
       ifMachineUrl must not be included in the request. For more information, see ifMachineUrl example below.
 
   -P, --parentCallId=parentCallId
-      The ID of the parent Call in the case that this new Call is meant to be treated as a child of an existing Call. This 
-      attribute should be included when possible to reduce latency when adding child calls to Conferences containing the 
-      parent Call. A call can only be used as a parent once the call is in progress or as an inbound call that is still 
-      ringing.  An outbound call is considered to be in progress once the outdialConnect or outdialApiConnect webhook is 
-      invoked.  An inbound call is ringing when the inbound webhook is invoked.
+      Required if no applicationId or callConnecturl have been provided. The ID of the parent Call in the case that this 
+      new Call is meant to be treated as a child of an existing Call. This attribute should be included when possible to 
+      reduce latency when adding child calls to Conferences containing the parent Call. A call can only be used as a 
+      parent once the call is in progress or as an inbound call that is still ringing.  An outbound call is considered to 
+      be in progress once the outdialConnect or outdialApiConnect webhook is invoked.  An inbound call is ringing when the 
+      inbound webhook is invoked. If a callConnectUrl attribute is also included with the parentCallId in the request, 
+      this URL will be used as a replacement of the callConnectUrl originally assigned in the parent call.
+
+  -c, --callConnectUrl=callConnectUrl
+      The URL that FreeClimb should use to handle this phone call. If an applicationId or parentCallId have already been 
+      provided, this callConnectUrl attribute will be used as a replacement of the callConnectUrl originally assigned in 
+      the application or parent call. See applicationId v. callConnectUrl below.
```

Comment:
> See applicationId v. callConnectUrl below.

???

Reply:
I didn't see any section called any variation of "applicationId v. callConnectUrl" in these docs, so I'm confused by this sentence.

Reply:
i removed all sentences that have the words see below in them as long as they dont have the word note as well